internal/notify/email.go

```diff
@@ -191,3 +193,15 @@ func (e PaymentConfirmationEmail) emailID(isProduction bool) string {
 
 	return "ff757818-f066-4605-8751-af481afe8a2b"
 }
+
+type AttorneyOptedOutEmail struct {
```

Comment:
I assume this will have similar details to the certificate provider one when it is done, so I'm just leaving these fields in